File: EstruturaDeDados&Algoritmos/B-Arrays/OutrasLinguagens/Compiladas/Array.cpp
#include <iostream>
#include <vector>

int main() {
    // Array estilo C (tamanho fixo)
    int numeros[] = {10, 20, 30};
    
    // Moderno (Vector - dinâmico)
    std::vector<int> vetor = {10, 20, 30};

    std::cout << numeros[0] << std::endl; // Imprime 10
    return 0;
}

File: EstruturaDeDados&Algoritmos/B-Arrays/OutrasLinguagens/Compiladas/Array.go
package main
import "fmt"

func main() {
    // Array de tamanho fixo [3]
    var numeros = [3]int{1, 2, 3}

	// Ou Declaração de Slice (mais comum que array fixo)
	numerosSlice := []int{10, 20, 30}
    
    fmt.Println(numeros) // Imprime [10 20 30]
    fmt.Println(numerosSlice)
}